src/runnner/utils.rs

```diff
@@ -0,0 +1,139 @@
+use clarity_repl::clarity::types;
+use clarity_repl::clarity::util::hash;
+use std::fmt::Write;
+
+pub fn value_to_string(value: &types::Value) -> String {
```

Comment:
I see this is just moved from another file, but is this function necessary? `Value` already implements `Display`. Should we open an issue to remove this and use that instead, or am I missing something that is different here?

Reply:
`Value` does implement `Display` however the formatting implemented in this version differs from the canonical implementation. The outcome of revisiting the canonical implementation (in the stacks-node / clarity-lib repo) could have some undesired side effects in a few places, so having this implementation might be more prudent.